endpoints/openrtb2/firstpartydata/first_party_data.go

```diff
@@ -0,0 +1,238 @@
+package firstpartydata
+
```

Comment:
Perhaps I missed this here but I'm not sure if this logic has been implemented:

If the bidder is not defined as having global permissions (`ext.prebid.data.bidders` exists and this bidder is not listed in it):
1. Remove sensitive FPD data (flagged as in-scope for permissions in the reqts) 
    - Delete `{app/site/user}.ext.data`
    - Delete `user.data`
    - Delete `{app/site}.content.data`
2. Remove FPD meta
    - Delete `ext.prebid.data.bidders`
    - Delete `ext.prebid.bidderconfig`

Reply:
`Delete user.data` - this is done with exception when this data can be unmarshalled to existing openrtb user.data.
`Delete {app/site}.content.data` - this is not done, was it in requirements? What to do with this data if it's present?
Everything else is done.

Reply:
Should `{app/site}.content.data` have a format of already defined openrtb.{app/site}.content.data?

Reply:
From discussion with @bretg and @mansinahar: Should bidder be defined in both `ext.prebid.data.bidders` and `ext.prebid.bidderconfig.ortb2` to receive `user.data`, `site/app.content.dat`a and `user.app/site.ext.data`?